packages/metadata-editor/src/MetadataEditor.tsx

```diff
@@ -304,17 +304,17 @@ export class MetadataEditor extends ReactWidget {
   }
 
   renderField(fieldName: string): React.ReactElement {
-    const uihints = this.schema[fieldName].uihints;
+    let uihints = this.schema[fieldName].uihints;
```

Comment:
My changes in this file do exactly that. Currently the MetadataEditor does not populate fields without a `uihints` This updates it so if `uihints` isn't in the schema for the specific fields then it creates a default label using the filed name. This change is not needed for Runtimes since I add `uihints` to the kfp schema in this PR, but it does fix the missing functionality we meant to include previously